.ci/Jenkinsfile

```diff
@@ -178,6 +179,40 @@ pipeline {
           githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
         }
       }
+      stage('Release') {
+        options { skipDefaultCheckout() }
+        environment {
+          RUBY_DOCKER_TAG = 'ruby:2.6'
+        }
+        when {
+          beforeAgent true
+          anyOf {
+            tag pattern: 'v\\d+.*', comparator: 'REGEXP'
+            expression { return params.Run_As_Master_Branch }
```

Comment:
Use some other kind of mechanism. `Run as master` is sometimes used for benchmarking PRs
```suggestion
```